lib/medium-admin.php

```diff
@@ -294,7 +358,13 @@ public static function cross_post($post, $medium_post, $medium_user) {
       "Accept-Charset" => "utf-8"
     );
 
-    $response = wp_remote_post("https://api.medium.com/v1/users/" . $medium_user->id . "/posts", array(
+    if ($medium_post->publication_id != NO_PUBLICATION) {
```

Comment:
Should NO_PUBLICATION be used as the default above ('Default to none'), rather than user->default_publication_id?